src/ethereum_test_tools/exceptions/exceptions.py

```diff
@@ -147,6 +147,10 @@ class BlockException(ExceptionBase):
     """
     Block's excess blob gas in header is incorrect.
     """
+    RLP_ENCODING = auto()
```

Comment:
I think `RLP_ENCODING` should replace the existing `INCORRECT_BLOCK_FORMAT`, unless there's a block formatting error that does not result from an RLP encoding issue?

Reply:
You right this sound like an exception related to rlp itself.

1) rlp invalid - RLP_ENCODING  (currently there are no such tests)
2) eth structure invalid (failure to read rlp into eth structures)